code/datums/gamemodes/nuclear.dm

```diff
@@ -48,13 +48,21 @@
 
 		if (player.ready)
 			num_players++
+#ifndef ME_AND_MY_40_ALT_ACCOUNTS
+	if (num_players < 15)
+		boutput(world, SPAN_ALERT("<b>ERROR: Minimum player count of 15 required for Nuclear game mode, aborting nuke round pre-setup.</b>"))
```

Comment:
recommend logging this instead; the game will re-roll mode automatically now iirc?